### DISCUSSION TITLE
什么时候可以支持安卓14

### DISCUSSION THREAD

**Discussion by Ivanzhanglm** [getlantern/lantern-forum-cn#121] [Q&A]
安卓14系统下提示不兼容当前软件版本，请问什么时候可以更新？

**Comment by MaxMKMax** [comment]
@Ivanzhanglm 您好, 是的。目前最新的蓝灯app不支持android 14。让我向技术团队咨询一下解决方案。对于造成的不便，我们深表歉意。
